app/src/main/res/layout/activity_reader.xml

```diff
@@ -26,7 +26,7 @@
                 android:elevation="8dp"
                 android:minHeight="@dimen/standard_toolbar_height"
                 android:theme="@style/Theme.AppCompat"
-                app:layout_scrollFlags="scroll|enterAlways"></android.support.v7.widget.Toolbar>
```

Comment:
Oh yeah I did this in ONE spot too because I saw the linter complaining with my own eyes. Nothing changed. It just won't be angry you had an empty node.
This happens in other spots too. I just only fixed it here for some reason.